.changeset/bright-mayflies-remain.md

```diff
@@ -0,0 +1,13 @@
+---
+"perseus-build-settings": patch
+"@khanacademy/kas": patch
+"@khanacademy/kmath": patch
+"@khanacademy/math-input": patch
+"@khanacademy/perseus-editor": patch
+"@khanacademy/perseus-error": patch
+"@khanacademy/perseus-linter": patch
+"@khanacademy/pure-markdown": patch
+"@khanacademy/simple-markdown": patch
```

Comment:
I'm surprised by these all needing a patch change.  I would have expected only `@khanacademy/perseus` would show up in this list for this PR.